assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.js

```diff
@@ -70,18 +75,83 @@ export default function SettingsEdit() {
 
 		return false;
 	} );
+	const publicationID = useSelect( ( select ) =>
+		select( MODULES_READER_REVENUE_MANAGER ).getPublicationID()
+	);
+	const publicationAvailable = useSelect( ( select ) => {
+		if ( hasModuleAccess === undefined ) {
+			return undefined;
+		}
+
+		if ( hasModuleAccess === false ) {
+			return false;
+		}
+
+		const publications = select(
+			MODULES_READER_REVENUE_MANAGER
+		).getPublications();
+
+		if ( ! Array.isArray( publications ) ) {
+			return undefined;
+		}
+
+		return publications.some(
+			// eslint-disable-next-line sitekit/acronym-case
+			( { publicationId: id } ) => id === publicationID
+		);
+	} );
+	const formattedOwnerName = useSelect( ( select ) => {
+		const module = select( CORE_MODULES ).getModule(
+			READER_REVENUE_MANAGER_MODULE_SLUG
+		);
+
+		return module?.owner?.login
+			? `<strong>${ module.owner.login }</strong>`
+			: __( 'Another admin', 'google-site-kit' );
+	} );
 
 	if ( isDoingSubmitChanges || undefined === hasModuleAccess ) {
 		return <ProgressBar />;
 	}
 
 	return (
 		<div className="googlesitekit-setup-module googlesitekit-setup-module--reader-revenue-manager">
-			{ hasModuleAccess && (
-				<Fragment>
-					<PublicationSelect hasModuleAccess={ hasModuleAccess } />
-					<PublicationOnboardingStateNotice />
-				</Fragment>
+			{ hasModuleAccess && false === publicationAvailable && (
+				<ErrorText
+					message={ sprintf(
+						/* translators: 1: Publication ID. */
+						__(
+							'The previously selected publication with ID %s was not found. Please select a new publication.',
+							'google-site-kit'
+						),
+						publicationID
+					) }
+				/>
+			) }
+			<div className="googlesitekit-setup-module__inputs">
```

Comment:
Added this for proper spacing around the publication select field. This is consistent with settings edit screens of other modules, e.g. Analytics.